store/localstore/kv.go

```diff
@@ -155,16 +114,11 @@ func (s *dbStore) scheduler() {
 				continue
 			}
 			switch cmd.op {
-			case opSeek:
-				seekCh <- cmd
 			case opCommit:
 				s.doCommit(cmd)
 			}
```

Comment:
Former implementation guarantees that _Commit_ is an atomic operation -- no _Read_ can be mixed in _Commit_, otherwise it will lead to _Dirty Read_.

Consider a write transaction is committing with a ts _T1_, and a concurrent transaction is reading with a greater ts _T2_. The result depends on which one access engine first rather than their timestamps.

I guess this is the issue @qiuyesuifeng mentioned.